hips/hip-0012.md

```diff
@@ -1,38 +1,138 @@
 ---
 hip: "0012"
 title: "Helm 4 Development Process"
-authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+authors: [ "Matt Butcher <matt.butcher@fermyon.com>", "George Jenkins <gvjenkins@gmail.com>", "Matt Farina <matt.farina@suse.com>" ]
 created: "2021-04-02"
 type: "process"
 status: "draft"
 ---
 
 ## Abstract
 
-The Helm 4 development lifecycle will be managed by a release engineer.
-It will be kicked off in October, 2021.
-The process will last not less than six months.
-There will be an active development phase, followed by an alpha period, a beta period, and a release candidate period.
-The release manager will determine when Helm 4 is ready to ship.
+Helm 4 is planned as the next major version release of Helm.
+With KubeCon/CloudNativeCon North America 2024 marking the fifth anniversary of the successful Helm 3 release, the development of Helm 4 is strongly desired in order to continue evolving Helm and its ecosystem.
```

Comment:
hmm, one goal was to point out that it has been a long time since Helm 3's release. I guess I can just refer to KubeCon 2019 as the release of Helm 3, and point out that has been at least 5 years since